### DISCUSSION TITLE
[Script request] Funkwhale

### DISCUSSION THREAD

**Discussion by rlimaeco** [community-scripts/ProxmoxVE#59] [Request script]
### Application Name

Funkwhale

### Website

https://www.funkwhale.audio/

### Description

Funkwhale is a self-hosted audio player and publication platform. It enables users to build libraries of existing content and publish their own.

Funkwhale uses the [ActivityPub protocol](https://www.w3.org/TR/activitypub/) to talk to other apps across the [Fediverse](https://docs.funkwhale.audio/glossary.html#term-Fediverse). Users can share content between [Funkwhale pods](https://docs.funkwhale.audio/glossary.html#term-Pod) or with other Fediverse software.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by itsdoublearon** [comment]
I to would like to see this added please.
